plugins/faucet/params.go

```diff
@@ -0,0 +1,44 @@
+package faucet
+
+import (
+	"time"
+
+	flag "github.com/spf13/pflag"
+
+	"github.com/gohornet/hornet/pkg/node"
+	iotago "github.com/iotaledger/iota.go/v2"
+)
+
+const (
+	// the amount of funds the requester receives if the target address has less funds than the given amount.
+	CfgFaucetAmount = "faucet.amount"
+	// the amount of funds the requester receives if the target address has more funds than the given amount.
+	CfgFaucetSmallAmount = "faucet.smallAmount"
+	// the maximum allowed amount of funds on the target address.
+	CfgFaucetMaxAddressBalance = "faucet.maxAddressBalance"
+	// the maximum output count per faucet message.
+	CfgFaucetMaxOutputCount = "faucet.maxOutputCount"
```

Comment:
why is this configurable? to avoid big pow?

Reply:
I can't remember but I just say yes :trollface: